data-raw/ni-beds.R

```diff
@@ -0,0 +1,44 @@
+# Northern Ireland on Health Inequalities Explorer
+# Secondary care indicators - bed availability
+
+# ---- Load libs ----
+library(tidyverse)
+library(devtools)
+library(lubridate)
+
+# ---- Load internal sysdata.rda file with URL's ----
+load_all(".")
+
+query_url <-
+  query_urls |>
+  filter(id == "ni_beds") |>
+  pull(query)
+
+# ---- Download and wrangle data ----
+raw <- read_csv(query_url, skip=2)
+
+# Change dataset to tidy data with all relevant information
+# Removed "Day Case" column since it included missing values and duplicate values with "Total Day Cases".
+# Can calculate number of discharge beds = total available beds - total occupied beds
```

Comment:
Where possible, we do not want to perform our own calculations in the humaniverse data packages, and try and ship the data as close to its raw form as possible. Calculations can be added in the projects where they are used.

Reply:
Yes, that makes sense. Would you like me to remove the discharged beds information and then try to find a separate dataset for that information?

Reply:
I'm just going to finish the code review now with some more comments 👍

Reply:
> Yes, that makes sense. Would you like me to remove the discharged beds information and then try to find a separate dataset for that information?

Yes, please remove this compute variable, and add a separate data set if it exists. However, I don't this data is available after our previous team discussions last week.